### PR TITLE
Make holster verb universal for accessory/pouch holsters, fix holster oversights

### PR DESCRIPTION
**Pull request by ErisMeleeEnjoyer**
## About The Pull Request

made most guns that should be holsterable... holsterable (i checked every gun manually)
Accessory holsters cannot hold stuff that pouch holsters can't
made the holster hotkey a human verb, letting it access both types of holsters and interact with them, making the hotkey universal(it also doesn't break if you have both types of holsters) 

## Why It's Good For The Game

i was blackmailed

god why
## Changelog
:cl: Kegdo
add: holster verb(hotkey) is universal for both holster types
fix: Most holsterable guns should be holsterable now
fix: Accessory holsters cannot hold stuff that pouch holsters can't
/:cl:

### CODE REVIEW

**Comment by ErisMeleeEnjoyer** [comment]
hey @SirRichardFrancis check this out

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -1583,6 +1583,36 @@ var/list/rank_prefix = list(\
 	dodging = !dodging
 	to_chat(src, "<span class='notice'>You are now [dodging ? "dodging incoming fire" : "not dodging incoming fire"].</span>")
 	return
+
+/mob/living/carbon/human/verb/access_holster()
+	set name = "Holster"
+	set desc = "Try to access your holsters."
+	set category = "IC"
+	if(stat) return
```
Comment (SirRichardFrancis) [inline code/modules/mob/living/carbon/human/human.dm:1591]: Guidelines.
```suggestion
	if(stat)
		return
```

File: code/modules/mob/living/carbon/human/human.dm [hunk 2]
```diff
@@ -1583,6 +1583,36 @@ var/list/rank_prefix = list(\
...
+					if(get_active_hand())//do we hold something?
+						H.attackby(get_active_hand(), src)
+					else
+						H.attack_hand(src)
+					holster_found = TRUE
+					return
+	//nothing at all!
+	if(!holster_found)
+		to_chat(src, "<span class='notice'>You don\'t have any holsters.</span>")
+	return
```
Comment (SirRichardFrancis) [inline code/modules/mob/living/carbon/human/human.dm:1614]: Redundant.
```suggestion
```

File: code/modules/mob/living/carbon/human/human.dm [hunk 3]
```diff
@@ -1583,6 +1583,36 @@ var/list/rank_prefix = list(\
...
 	return
+
+/mob/living/carbon/human/verb/access_holster()
+	set name = "Holster"
+	set desc = "Try to access your holsters."
+	set category = "IC"
+	if(stat) return
+	var/holster_found = FALSE
+
+	for(var/obj/item/storage/pouch/holster/holster in list(back, s_store, belt, l_store, r_store,))
```
Comment (SirRichardFrancis) [inline code/modules/mob/living/carbon/human/human.dm:1594]:
```suggestion
	for(var/obj/item/storage/pouch/holster/holster in list(back, s_store, belt, l_store, r_store))
```

**Review by SirRichardFrancis — COMMENTED** [review]
Alright, after looking at this for a while, I have no more complaints.

**Review by SirRichardFrancis — APPROVED** [review]

File: code/game/objects/items/weapons/storage/holster.dm [hunk 4]
```diff
@@ -42,9 +44,18 @@
...
+		/obj/item/gun/projectile/boltgun/levergun/sawn,
+		/obj/item/gun/projectile/automatic/slaught_o_matic,
+		/obj/item/gun/energy/rxd,
+		/obj/item/gun/energy/nuclear,
+		/obj/item/gun/energy/nt_svalinn,
+		/obj/item/gun/energy/crossbow,
+		/obj/item/reagent_containers/food/snacks/mushroompizzaslice,
+		/obj/item/reagent_containers/food/snacks/meatpizzaslice,
+		/obj/item/reagent_containers/food/snacks/vegetablepizzaslice,
+		/obj/item/bananapeel
```
Comment (TheShown911) [inline code/game/objects/items/weapons/storage/holster.dm:58]: Based?

File: code/modules/mob/living/carbon/human/human.dm [hunk 5]
```diff
@@ -1583,6 +1583,36 @@ var/list/rank_prefix = list(\
...
+				for(var/obj/item/clothing/accessory/holster/H in U.accessories)
+					if(get_active_hand())//do we hold something?
+						H.attackby(get_active_hand(), src)
+					else
+						H.attack_hand(src)
+					holster_found = TRUE
+					return
+	//nothing at all!
+	if(!holster_found)
+		to_chat(src, "<span class='notice'>You don\'t have any holsters.</span>")
```
Comment (TheShown911) [inline code/modules/mob/living/carbon/human/human.dm:1614]: Use SPAN_NOTICE macro.

**Review by TheShown911 — COMMENTED** [review]
What if it just checked the holsterable flag on guns?

**Comment by ErisMeleeEnjoyer** [comment]
> What if it just checked the holsterable flag on guns?

Naw. Most guns don't have that flag, and it's basically a bay leftover that should just be removed at this point.

**Review by TheShown911 — APPROVED** [review]
It's much cleaner to just add a tag on a gun, rather than constantly update 2 lists when any new weapon is introduced, but you do you.

**Comment by ErisMeleeEnjoyer** [comment]
I do you m-

**Pull request merged**